modules/javafx.graphics/src/main/native-prism-es2/ios/IOSWindowSystemInterface.m

```diff
@@ -82,7 +82,9 @@ jboolean clearCurrentContext(void *context) {
 }
 
 jboolean deleteContext(void *context) {
-    fprintf(stderr, "IOSWindowSystemInterface : deleteContext unimp\n");
+    if (pulseLoggingRequested) {
+        fprintf(stderr, "IOSWindowSystemInterface : deleteContext unimp\n");
```

Comment:
this doesn't relate to pulseLogging I believe?

Reply:
Right, but since it wasn't commented out, should I just leave it as it was?

Reply:
@johanvos  I've finally restored the initial printout.